lib/themes/dosomething/paraneue_dosomething/js/components/Button.js

```diff
@@ -0,0 +1,37 @@
+const React = require('react');
+
+/**
+ * Button Component
+ * <Button />
+ */
+const Button = (props) => {
+    const classes = ['button'].concat(props.classes);
```

Comment:
But that class should be applied based on a prop, like any of the others. It shouldn't be the `FeatureCard`'s concern what the actual classname is.